### ISSUE TITLE
Crop timers taking fertilize into account

### ISSUE THREAD

**Issue by DenL** [Ottermandias/Accountant#13]
The crop timers currently track when a crop/plant would mature, but does not take fertilizing into acccount. Possible to track fertilizing as well (decreasing mature time by 1 hour)?

Thanks!

**Comment by DenL** [comment]
I was under the impression that's precisly how it work and it alligns with my past experiences.
E.g. I plant this crop at 4pm but I want to be able to harvest it a week later at 7am due to IRL schedule, I would fertilize it 9+ times during the week and it would mature before 7am. Sometimes you can even see crops cross over growth stages right after you fertilize.

if you remain unconvinced, perhaps you could consider just a tracker to see how many times you've fertilized a plant (just a counter in tooltip, does not affect timer)? Thanks.

**Comment by Ottermandias** [comment]
Yes, fertilizing updates the timer, which may cause crops that should be already grown out but are not due to ward update times to update and be finished.
Otherwise it is very probably based on the time remaining, i.e. later fertilizations reduce the remaining time for less than earlier fertilizations.

**Comment by spval** [comment]
https://www.ffxivgardening.com/fertilizer-analysis
